src/css/base.styl

```diff
@@ -1528,6 +1528,10 @@ nav ul
             .strength
               transform(rotate(90deg))
 
+            .subtype-target
+              transform(rotate(90deg), translateX(100%))
```

Comment:
This rotates it the wrong way for the runner... missing the corresponding code under `.corp-board.opponent` to rotate / translate it back. `transform(rotate(-90deg), translateY(-100%))` seems to do it for me.

Comparison:
Old
<img width="289" alt="screen shot 2016-09-23 at 19 47 19" src="https://cloud.githubusercontent.com/assets/13198563/18796025/b4a60ed8-81c7-11e6-91f0-a38673d9a3e1.png">
New
<img width="288" alt="screen shot 2016-09-23 at 19 54 03" src="https://cloud.githubusercontent.com/assets/13198563/18796036/b9c80a24-81c7-11e6-945f-84d2a095fa10.png">